src/communication/azext_communication/action.py

```diff
@@ -10,7 +10,6 @@
 # pylint: disable=wildcard-import
 # pylint: disable=unused-wildcard-import
 
-from .generated.action import *  # noqa: F403
```

Comment:
so we don't need to replace this import with the new generated actions?

Reply:
There's no equivalent for the new code. Everything that needs to be imported for the new generated code is under the init file